modular_robot/revolve2/modular_robot/sensor_state/_drone_sensor_state.py

```diff
@@ -0,0 +1,19 @@
+from abc import ABC, abstractmethod
+
+from ..body.base import MotorSensor
+from ._motor_sensor_state import MotorSensorState
+
+
+class DroneSensorState(ABC):
+    """The state of modular robot's sensors."""
+
+    @abstractmethod
+    def get_active_hinge_sensor_state(
```

Comment:
i think you should change this to something with your motors :D

Reply:
Yes, this shouldn't be here. Don't need sensors for the motors.